blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { Carousel, Image, Icon } from "@wpmedia/arc-themes-components";
+import { useContent } from "fusion:content";
+import { extractImageFromStory } from "@wpmedia/resizer-image-block";
+
+// import { useFusionContext, useAppContext } from "fusion:context";
+// import getProperties from "fusion:properties";
+// import getThemeStyle from "fusion:themes";
+// import getTranslatedPhrases from "fusion:intl";
+const BLOCK_CLASS_NAME = "b-story-carousel";
+
+const StoryCarousel = ({
+	arcSite,
+	customFields: {
+		carouselContentConfig: { contentService = "", contentConfigValues = {} } = {},
+		headline,
+		carouselLabel = "Carousel of Images",
+		identifier,
```

Comment:
For the identifier we can switch this from a custom field and use the id returned in the `useComponentContext()` hook

```
import { useComponentContext } from "fusion:context";

```